lib/parallel_guest_migration_base.pm

```diff
@@ -570,12 +570,18 @@ sub config_host_shared_storage {
             }
             $_temppath = $args{exppath};
             $_temppath =~ s/\//\\\//g;
+            # Create export path directory if it doesn't exist
+            unless (-d $_temppath) {
+                assert_script_run("mkdir -p $_temppath");
+            }
+            assert_script_run("cp -r $args{mntpath}/* $args{exppath}/");
```

Comment:
Maybe good to add a comment to explain why this is needed (sth like guest installation puts images to xx by default, move them to nfs export directory before migration, ... ).

BTW, will mv be better than cp? To avoid confusion what image is used when there is problem in nfs.